filter_plugins/oo_filters.py

```diff
@@ -18,9 +18,9 @@
 from distutils.util import strtobool
 from distutils.version import LooseVersion
 from operator import itemgetter
-from ansible.module_utils.six.moves.urllib.parse import urlparse
 from ansible.parsing.yaml.dumper import AnsibleDumper
 from six import string_types
+from six.moves.urllib.parse import urlparse
```

Comment:
Filter plugins run from the command host, might as well use six rather than ansible.module_utils.six.